paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -738,3 +738,8 @@ export const getAuditLogByDivId = (divId: DivId): AuditLog | undefined => {
   }
   return getAuditLogByTransaction(prebidTransactionId);
 };
+
+export const deleteIdsAndPreferences = (): Promise<boolean> => {
```

Comment:
missing `options` parameter, to pass the proxy hostname

Reply:
done.